docs/user-guide/cli-configure-scs-on-zlinux-os.md

```diff
@@ -0,0 +1,53 @@
+# Configure Secure Credential Store on z/Linux operating systems
+
+The Secure Credential Store (SCS) does not contain the  native, pre-built binaries that are required to access the credential vault on z/Linux operating systems.
+
+Because the credential manager is now a built-in function of Zowe CLI, developers must build the credential mananger binaries on z/Linux systems during the Zowe CLI installation process.
+
+The following steps describe how to install and build the credential store binaries on z/Linux (Red Hat Enterprise Linux (RHEL) and Ubuntu) systems.
+
+1. Install the following Linux packages on the z/Linux system:
+    - make
+    - gcc-c++ (sometimes available as g++)
+    - gnome-keyring
+    - libsecret (sometimes available as libsecret-1-0)
+    - libsecret-devel (sometimes available as libsecret-1-dev)
+    - Python 3.6 or later
+
+    **Note:** If you are installing the Linux packages on a RHEL system, the system where you are configuring SCS might require Internet access. When a site hosts its own package repositories, the repositories might not contain all of the packages that are required to configure the SCS. Under this scenario, the RHEL system requires Internet access to install the required packages.
```

Comment:
This isn't specific to a RHEL system, if you are installing Linux packages, any system might require internet access.